### PR TITLE
Kirkwood (dockstar) support

### PR DESCRIPTION
**Pull request by thypon**
I take some of the work done in archlinuxarm to boot these boards leaving the attribution.

### CODE REVIEW

File: srcpkgs/rpcbind/template [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 version=0.2.3
-revision=2
+revision=3
 build_pie=yes
 build_style=gnu-configure
 configure_args="--enable-warmstarts --with-statedir=/run --with-rpcuser=rpc
  $(vopt_if systemd '--with-systemsystemunitdir=/usr/lib/systemd/system' '--with-systemdsystemunitdir=no')"
 hostmakedepends="pkg-config"
-makedepends="libtirpc-devel $(vopt_if systemd systemd-devel)"
+makedepends="libtirpc-devel mit-krb5-devel $(vopt_if systemd systemd-devel)"
```
Comment (xtraeme) [inline srcpkgs/rpcbind/template]: This change makes no sense:

- hardcoding `/usr/include` will make cross compilation fail
- you are appending to CFLAGS incorrectly.

File: srcpkgs/iw/template [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-revision=1
+revision=2
 build_pie=yes
 build_style=gnu-makefile
 make_install_args="SBINDIR=/usr/bin"
-hostmakedepends="pkg-config libnl3-devel"
-makedepends="libnl3-devel"
-short_desc="nl80211 based CLI configuration utility for wireless devices"
+hostmakedepends="pkg-config"
+makedepends="libnl-devel"
```
Comment (xtraeme) [inline srcpkgs/iw/template]: This change is also wrong, switching to the libnl 1.x is not the proper fix.

File: srcpkgs/rpcbind/template (hunk 1 above)
Comment (thypon) [inline srcpkgs/rpcbind/template]: the CFLAGS append isn't needed anymore :-S i forgot here.

File: srcpkgs/iw/template (hunk 2 above)
Comment (xtraeme) [inline srcpkgs/iw/template]: I just fixed iw, please remove that change.

**Comment by xtraeme** [comment]
Please remove the changes that are unrelated to this pull request (iw and rpcbind).

**Comment by thypon** [comment]
Done, they where the fixes i've done to compile all the needed packages for my kirkwood root.

File: srcpkgs/dockstar-uboot/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'kirkwood-uboot'
+_board=dockstar # It's easy to add also the other kirkwood boards replacing this with "goflexhome goflexnet pogo_e02"
+pkgname="${_board}-uboot"
+version=2014.04.R4
+revision=3
```
Comment (xtraeme) [inline srcpkgs/dockstar-uboot/template]: `revision=1` for new pkgs please.

Comment (thypon) [inline srcpkgs/dockstar-uboot/template]: here we are

**Comment by thypon** [comment]
better patch handling.

**Pull request merged**